ts/src/header-validator/validate-json.ts

```diff
@@ -1576,9 +1576,9 @@ function reportDestination(ctx: Context, j: Json): Maybe<string | string[]> {
         maxLength: 3,
       }).filter((v) => {
         for (let i = 1; i < v.length; ++i) {
-          if (v[i]! < v[i - 1]!) {
+          if (v[i]! <= v[i - 1]!) {
```

Comment:
Not related to this PR, but looks like warning is reported for`https://A.test`: URL components other than site (https://a.test) will be ignored

Reply:
I can look into that separately.